scripts/main.js

```diff
@@ -62,6 +59,21 @@ var updateDOM = function(responseJSON) {
       ' - [ <a href="http://maps.google.com/?q=' + location.name +
       '" target="_blank">Directions</a> ]'
     );
+
+  // Add embed map
+  var map = document.getElementById('map');
+  map.src = 'https://www.google.com/maps/embed/v1/place?' +
+    'key=AIzaSyC9JxGGhS0Xr1429ae62E3MEfk227kqDIA&q=' +
+    location.address_1;
```

Comment:
Use `location.name` here instead, it seems to render a more accurate result on the map. Perhaps postfix Cork too. i.e..

```js
map.src = 'https://www.google.com/maps/embed/v1/place?' +
  'key=AIzaSyC9JxGGhS0Xr1429ae62E3MEfk227kqDIA&q=' +
  location.name + ', Cork, Ireland';
```